### PR TITLE
add information about disabled monitoring in rm view

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringHostView.java [hunk 1]
```diff
@@ -210,7 +210,8 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        status.setContents("Monitoring is unavailable on this host. Maybe node was started with --disableMonitoring option.<br>"+
```
Comment (fviale) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringHostView.java]: Monitoring is unavailable on this host. Either monitoring was disabled with the disableMonitoring option or an error occurred. More information are available inside the node logs.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringNodeView.java [hunk 2]
```diff
@@ -180,7 +180,8 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        status.setContents("Monitoring is unavailable on this node. Maybe node was started with --disableMonitoring option.<br>"+
+                "Error message: "+ JSONUtils.getJsonErrorMessage(caught));
```
Comment (fviale) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringNodeView.java]: Monitoring is unavailable on this node. Either monitoring was disabled with the disableMonitoring option or an error occurred. More information are available inside the node logs.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringSourceView.java [hunk 3]
```diff
@@ -192,7 +192,11 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        String errorMessage = JSONUtils.getJsonErrorMessage(caught);
+        if(errorMessage.contains("HTTP 500")){
+            errorMessage = "Monitoring is not available on this node source";
```
Comment (fviale) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringSourceView.java]: I suppose monitoring cannot be disabled for a node source. In that case it is clearly an error. What about : Monitoring is not available on this node source because of an internal error. More information are available inside the scheduler logs.

Comment (ShatalovYaroslav) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringSourceView.java]: You are right. In fact this change is not related to disabling monitoring. Sorry for mess. It was added to this pull request automatically when the commit was pushed. I noticed that usually we have "HTTP 500 Internal Server Error", when click on monitoring tab for Source Node. The aim of this commit to show proper message for this case.

Comment (ShatalovYaroslav) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringSourceView.java]: In most cases the Node Source monitoring is not available
